hadoop-tools/hadoop-aws/src/test/java/org/apache/hadoop/fs/s3a/ITestS3APrefetchingInputStream.java

```diff
@@ -94,17 +97,16 @@ public void teardown() throws Exception {
     largeFileFS = null;
   }
 
-  private void openFS() throws Exception {
+  private void openFS(String fileName) throws Exception {
```

Comment:
change to using the existing fs, and using methodPath() to generate a per test case path (no test uses more than one file, does it?)
```java

void createLargeFile() throws IOException {
  Configuration conf = getConfiguration();
  byte[] data = ContractTestUtils.dataset(S_1K * 72, 'x', 26);
  largeFile = methodPath();
  FileSystem largeFileFS = getFileSystem();
  ContractTestUtils.writeDataset(largeFileFS, largeFile, data, data.length, 16, true);
  FileStatus fileStatus = largeFileFS.getFileStatus(largeFile);
  largeFileSize = fileStatus.getLen();
  numBlocks = calculateNumBlocks(largeFileSize, BLOCK_SIZE);
}
```

Reply:
> no test uses more than one file, does it?

correct, they use only 1 file